apps/meteor/tests/end-to-end/api/24-methods.js

```diff
@@ -123,6 +126,276 @@ describe('Meteor.methods', function () {
 		});
 	});
 
+	describe('[@getReadReceipts]', () => {
+		it('should fail if not logged in', async () => {
```

Comment:
This request will fail even if the user is logged in, since no params are being passed.
You should send an otherwise valid request and also check the error type to ensure it's the user state that is causing the request to fail.